### ISSUE TITLE
Disable page border indicator

### ISSUE THREAD

**Issue by sati-bodhi** [OpenBoard-org/OpenBoard#834]
Is there a way to toggle the page border indicator on/off? 
![Screenshot from 2023-11-27 11-20-18](https://github.com/OpenBoard-org/OpenBoard/assets/7592695/d7461ff6-69fc-4560-82c2-b0077d6ef3b9)

**Comment by SeeDborg** [comment]
I agree with that, an option in the preferences to disable it would be great.
